account_financial_report/report/general_ledger.py

```diff
@@ -342,7 +342,7 @@ def _get_move_line_data(self, move_line):
             "rec_id": move_line["full_reconcile_id"][0]
             if move_line["full_reconcile_id"]
             else False,
-            "rec_name": move_line["full_reconcile_id"][1]
+            "rec_name": move_line["matching_number"]
```

Comment:
Instead of bringing another field, which is the same:

```suggestion
            "rec_name": str(move_line["full_reconcile_id"][0])
```